benchmarks/models.py

```diff
@@ -0,0 +1,112 @@
+import torch
+from torch.nn import Linear, ReLU, CrossEntropyLoss, Sequential, Conv2d, MaxPool2d, Softmax, Module, BatchNorm2d
+
+
+class Model_0(Module):
```

Comment:
Rename them to be a bit more informative -- e.g. BaseCNN and ExpandedCNN, or really anything that differentiates them.

Reply:
Yeah model 0 and 1 really aren't significantly different - I'm thinking about nixing 0 and having model 1 be our "base". Then we can adjust to no padding.